scripts/deploy-frontend.sh

```diff
@@ -22,12 +22,17 @@ update_frontend()
         echo "REACT_APP_BACKEND_URL=https://api-tanfdata.acf.hhs.gov/v1" >> .env.production
         echo "REACT_APP_BACKEND_HOST=https://api-tanfdata.acf.hhs.gov" >> .env.production
         echo "REACT_APP_LOGIN_GOV_URL=https://secure.login.gov/" >> .env.production
+        echo "REACT_APP_CF_SPACE=$CF_SPACE" >> .env.production
     else
-        echo "REACT_APP_BACKEND_URL=https://$CGHOSTNAME_BACKEND.app.cloud.gov/v1" >> .env.production
-        echo "REACT_APP_BACKEND_HOST=https://$CGHOSTNAME_BACKEND.app.cloud.gov" >> .env.production
+        echo "REACT_APP_BACKEND_URL=https://$CGHOSTNAME_BACKEND.app.cloud.gov/v1" >> .env.development
+        echo "REACT_APP_BACKEND_HOST=https://$CGHOSTNAME_BACKEND.app.cloud.gov" >> .env.development
+        echo "REACT_APP_CF_SPACE=$CF_SPACE" >> .env.development
```

Comment:
When in staging react scripts will use the `.env.development` file so variables for that environment need to be placed in that file.

Reply:
I recall having a conversation with Aaron that all deployments used `.env.production` which is a misnomer but what it is. I don't see where in this PR that may have changed. I'd like to verify that this is functional -- can we have a call or discuss during the tabletop?

Reply:
The react scripts additions I made to package.json in a previous ticket do change which .env is used. I am pretty convinced of this after my testing. Before I changed those values to `.env.development` the site was redirecting to the local host addresses, which are set in `.env.development`. Technically, these lines could be placed in there as well.

Reply:
I will document this in action.

Reply:
Here the main build script specifies which env variable to use `sh -ac '. ./.env.${REACT_APP_ENV};`

https://github.com/raft-tech/TANF-app/blob/release/v2.0.1/tdrs-frontend/package.json#L33

The lines after that set that variable.